feeds/wifi-trunk/hostapd/patches/900-hapd-netlink-ubus-bridge.patch

```diff
@@ -488,7 +493,7 @@
 +	}
 +	struct hostapd_event_avl_rec *rec = NULL;
 +	struct timespec ts;
-+	clock_gettime(CLOCK_REALTIME, &ts);
++	clock_gettime(CLOCK_MONOTONIC, &ts);
```

Comment:
Does this change align with Cloud side expectations?

Reply:
I don't think cloud would have any issues with this change. The wall clock(CLOCK_REALTIME) can jump forwards and backwards when the local clock is changed for example by NTP. MONOTONIC, on the other hand will make sure time always moves forward and thus is the best option when we are trying to calculate elapsed time between two events.

Reply:
I also looked at the code in the reference product and there also we are using monotonic clock.

Reply:
Since cloud is expecting events to be reported with clock time so that it can be aligned across systems, I have pushed a new commit reverting back to realtime